var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -379,8 +379,9 @@ def make_defs(self):
         if "+consistent_fpcsr" in self.spec:
             make_defs += ["CONSISTENT_FPCSR=1"]
 
-        # Flang/f18 does not provide ETIME as an intrinsic
-        if self.spec.satisfies("%clang"):
+        # Flang/f18 does not provide ETIME as an intrinsic.
+        # Do not set TIMER variable if fortran is disabled.
+        if self.spec.satisfies("%clang+fortran"):
```

Comment:
```suggestion
        if self.spec.satisfies("+fortran%clang"):
```

is a bit more clear imho